app/src/main/java/org/simple/clinic/summary/bloodsugar/BloodSugarSummaryViewEffectHandler.kt

```diff
@@ -2,15 +2,25 @@ package org.simple.clinic.summary.bloodsugar
 
 import com.spotify.mobius.rx2.RxMobius
 import io.reactivex.ObservableTransformer
+import org.simple.clinic.bloodsugar.BloodSugarRepository
+import org.simple.clinic.util.scheduler.SchedulersProvider
 
-class BloodSugarSummaryViewEffectHandler {
+object BloodSugarSummaryViewEffectHandler {
```

Comment:
We initially started with using `object` declarations for effect handlers, but that became too verbose over time, so we changed to declaring them as classes and providing the dependencies as class properties. This lets us inject the class when setting up the mobius loop as well.

Could you please change this to a class declaration? `SetupActivityEffectHandler` is a good example.

Note: it is true that some of our effect handlers are still declared as `object` types. We should work towards migrating them as well.

Reply:
Since the EffectHandlers are written differently in different places, I wasn't sure which one to follow. I will make this change.